bazel/markdown_to_html/BUILD.bazel

```diff
@@ -0,0 +1,55 @@
+load("@npm//@bazel/concatjs:index.bzl", "ts_library")
+load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
+load("//bazel/esbuild:index.bzl", "esbuild_esm_bundle")
+
+package(default_visibility = ["//bazel/markdown_to_html:__subpackages__"])
+
+esbuild_esm_bundle(
+    name = "bin",
+    entry_point = ":index.ts",
+    external = [
+        "marked",
+    ],
+    output = "bin.mjs",
+    platform = "node",
+    target = "es2022",
+    deps = [
+        ":markdown_to_html_lib",
+        "@npm//marked",
```

Comment:
AFAIK `marked` does support CommonJS. Should we just use that and not bundle?

Reply:
Done, I was just cargo-culting the structure of the manifest rule on auto-pilot.